src/resources/ble_esp32.cc

```diff
@@ -1894,14 +1915,23 @@ PRIMITIVE(get_value) {
 }
 
 PRIMITIVE(write_value) {
-  ARGS(BleReadWriteElement, element, Object, value, bool, with_response, bool, flush, bool, allow_retry)
+  ARGS(BleReadWriteElement, element, Blob, value, bool, with_response, bool, flush, bool, allow_retry)
 
   Locker locker(BleResourceGroup::instance()->mutex());
 
   if (!element->service()->device()) FAIL(INVALID_ARGUMENT);
 
+  uint16 mtu;
```

Comment:
Not sure it is really beneficial to have this as a 16 bit local, but it might be the style around here.

Reply:
done.